examples/with-eth-passkeys-galore/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # @turnkey/with-eth-passkey-signers
 
+## 0.2.0
```

Comment:
this is primarily for tracking on our side. think a changelog would be helpful in keeping track of changes over time